tools/reviewer/aa/commands/InitCommand.java

```diff
@@ -42,6 +44,9 @@
   public static Flag<String> startuposRepo =
       Flag.create("https://github.com/google/startup-os.git");
 
+  @FlagDesc(name = "global_registry_config", description = "Path to global_registry.prototxt")
```

Comment:
I think we can take the global registry directly from the repo itself, so no need for a flag.

Reply:
@oferb fixed in https://github.com/google/startup-os/pull/500/commits/e0915c92017ded4188e955585e1109bab2bd7176

now the path is fixed at `tools/reviewer/global_registry.prototxt`. However, I'm not sure relative to _which_ repo it should be appended so pls clarify

Reply:
The current repo.
The way it would work, is you would clone `startup-os`, and then run `aa init`.
So you would use whatever config is currently in `master`.